apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Select_id_ranges.sql

```diff
@@ -656,4 +656,4 @@ SELECT count(*) FROM ids_filtered_all;
 --  1539848
 -- (1 row)
 -- 
--- Time: 47612.129 ms
\ No newline at end of file
+-- Time: 47612.129 ms
```

Comment:
I'm going to get rid of these changes since I made them and I know that CMac is doing this for all of the scripts in another PR so they will conflict for whichever ones goes first.